tests/ted/test.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, fn, base, cfg, gold):
         
         extended_path = {
             'PATH': str(base.resolve() / subdir / 'bin') + os.pathsep + os.environ['PATH'],
-            'LD_LIBRARY_PATH': str(base.resolve() / subdir / 'bin' / 'lib'),
+            'LD_LIBRARY_PATH': (str(base.resolve() / subdir / 'bin' / 'lib') +
```

Comment:
`(str(base.resolve() / subdir / 'bin' / 'lib')`

I am not python guy, can someone explain what's going on here?

Reply:
It's path concatenation. It resolves to:
```
/home/<user>/.../MediaSDK/tests/results/bin/lib
````

Reply:
imho, `bin/lib` path does not make sense to me.

Reply:
msdk samples binaries are located in `MediaSDK/tests/results/bin`, libmfxhw.so is located in `MediaSDK/tests/results/bin/lib`

Reply:
At this point I am officially lost!

> Media SDK should be compiled and installed to `/opt/intel/mediasdk`

> We test non-installed binaries (test system copies them from "libmfx_folder").

> msdk samples binaries are located in MediaSDK/tests/results/bin, libmfxhw64.so is located in MediaSDK/tests/results/bin/lib

There are 3 (three, T-H-R-E-E) :) locations which you noted here and which are supposed to hold binaries which we test. Which one do we actually use? If we use all 3 then how all that actually works?

I can say how I envision the layout. There could be 2 approaches or both of them implemented at once:
1. We test non-installed binaries. In this case we need to know the location where the build had happened - i.e. the one called CMAKE_BINARY_DIR in cmake
2. We test installed binaries. In this case we need to know GnuInstallDirs layout, i.e. PREFIX, LIBDIR, BINDIR (and maybe few others).

IF we will have 1) we potentially can integrate these tests into `make test`. IF we will have 2) we have ability to verify the installed msdk version. Hence - both approaches are valuable and I would suggest to implement both.

Reply:
We copy binaries from "libmfx_folder" defined in JSON to `MediaSDK/tests/results/bin` and use them for testing.
The first version implied that we need to have whole media stack installed in `/opt/intel/mediasdk` to make it work. I think I need to replace
```
Media SDK should be compiled and installed to `/opt/intel/mediasdk`
```
with
```
Media stack should be installed to `/opt/intel/mediasdk`
```

Reply:
libmfx_folder in the JSON file does not point to /opt/intel/mediasdk. It points to non-installed build location. This gives a confusion.

> Media stack should

In this case you need to list ingredients of the media stack.

Reply:
I see. That's why I said:
> We test non-installed binaries (test system copies them from "libmfx_folder").
Do I need to point out that we test binaries from "libmfx_folder" in README?